content/en/docs/09/07/_index.md

```diff
@@ -15,7 +15,7 @@ In a sidecar pattern, the functionality of the main container is extended or enh
 
 * Different resource profiles, i.e. independent resource accounting and allocation
 * Clear separation of concerns at packaging level, i.e. no strong coupling between containers
-* Reusability, i.e. sidecar containers can be paired with numerous different "main" containers
+* Reusability, i.e., sidecar containers can be paired with numerous "main" containers
```

Comment:
Don't agree, that's German comma syntax.

Reply:
No, see above

Reply:
Where?